docs/example-react-router.mdx

```diff
@@ -133,8 +133,45 @@ test('rendering a component that uses useLocation', () => {
 
 ## Testing Library and React Router v5
 
-In React Router v5, you need to pass the history object as a whole to the Route
-component.
+Given your App uses the correct React Router v5 API.
+
+```jsx
+// app.js
+import React from 'react'
+import {Link, Route, Switch, useLocation} from 'react-router-dom'
+
+const About = () => <div>You are on the about page</div>
+const Home = () => <div>You are home</div>
+const NoMatch = () => <div>No match</div>
+
+export const LocationDisplay = () => {
+  const location = useLocation()
+
+  return <div data-testid="location-display">{location.pathname}</div>
+}
+
+export const App = () => (
+  <div>
+    <Link to="/">Home</Link>
+
+    <Link to="/about">About</Link>
+
+    <Switch>
+      <Route exact path="/" component={Home} />
+
+      <Route path="/about" component={About} />
+
+      <Route component={NoMatch} />
+    </Switch>
+
+    <LocationDisplay />
+  </div>
+)
+```
+
+In your tests, pass the history object as a whole to the Router component.
+**Note:** React Router v5 [only works with History v4](https://github.com/remix-run/history#documentation),
```

Comment:
Are we linking exactly what we want to link? because it looks like we're linking to the history package's docs. If that's what we want to link to, I think we should only make the "History" word a hyperlink.

Reply:
@MatanBobi We are. In the README's `Documentation` section, it specifies that

>Version 4 is used in React Router versions 4 and 5.

This sentence is what I want to reference to. Do you think we can use the Link with highlight feature of Chrome, something like this?

https://github.com/remix-run/history#documentation:~:text=Version%204%20is%20used%20in%20React%20Router%20versions%204%20and%205.

Reply:
Got it, thanks :)